editor/src/clj/editor/game_project.clj

```diff
@@ -230,7 +238,7 @@
                     (g/connect settings-node :raw-settings self :raw-settings)
                     (g/connect settings-node :meta-info self :meta-info)
                     (g/connect settings-node :resource-settings self :resource-settings)
-                    (g/connect settings-node :setting-errors self :setting-errors)
+                    (g/connect settings-node :setting-errors self :build-errors)
```

Comment:
`ga,e.project` doesn't have its own errors, so we use its child errors (`:setting-errors`) as its own